docs/reference/indices/put-mapping.asciidoc

```diff
@@ -59,7 +59,7 @@ It has the following format:
 PUT /{index}/_mapping/{type}
 { body }
 --------------------------------------------------
-
+// NOTCONSOLE
```

Comment:
I'd prefer to convert this to a working example. I've never noticed this feature, btw!